### DISCUSSION TITLE
morjs只能迁移没有依赖任何UI框架的微信小程序吗（比如我微信小程序使用了MiniProgramUI）不知道是否可以迁移

### DISCUSSION THREAD

**Discussion by 315574925** [eleme/morjs#28] [Q&A] [answered]
morjs只能迁移没有依赖任何UI框架的微信小程序吗（比如我微信小程序使用了MiniProgramUI）不知道是否可以迁移

**Comment by lyfeyaj** [comment, marked as answer]
> morjs只能迁移没有依赖任何UI框架的微信小程序吗（比如我微信小程序使用了MiniProgramUI）不知道是否可以迁移

可以迁移的，参考文档： [MorJS 使用社区组件库指南](https://mor.eleme.io/guides/advance/use-community-component)

**Comment by 315574925** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
